Goobi/install/config/messages_en.properties

```diff
@@ -1 +0,0 @@
-info = Goobi instance of CHANGEME CHANGEME. <br /> <br /> For support questions please ask: <a href="mailto:support@intranda.com">support@intranda.com</a>.
```

Comment:
Why was this deleted? It is OK to add new messages, but please don't remove existing ones